readme_azure_deployment_steps.md

```diff
@@ -0,0 +1,118 @@
+# Flask To-Do App Dockerized
```

Comment:
Big fan of having this split out so the main readme doesn't get overwhelmed, but it might be nice to add a link to it from the main readme so people notice that it exists!

You can do that with markdown like: `[click here to read the azure deployment readme](./readme_azure_deployment_steps.md)`